.github/workflows/build-docker.yml

```diff
@@ -0,0 +1,94 @@
+name: "Build Pipeline (Docker)"
+
+on:
+  push:
+    branches:
+      - jesse/pipeline-deploy
```

Comment:
Should we keep this or was it just to test?

Reply:
i'm gonna remove this right before merge 🙂